static_rdf_server/routes/ontology.py

```diff
@@ -100,7 +107,8 @@ async def put_ontology(request: web.Request) -> web.Response:  # noqa: C901
         with open(path, "w") as file:
             file.write(ontology_file)
 
-    return web.Response(status=204)
+    headers = MultiDict([(hdrs.LOCATION, f"{ontology_type}/{ontology}")])
```

Comment:
Url skal være absolutt.

The Location response-header field is used to redirect the recipient to a location other than the Request-URI for completion of the request or identification of a new resource. For 201 (Created) responses, the Location is that of the new resource which was created by the request. For 3xx responses, the location SHOULD indicate the server's preferred URI for automatic redirection to the resource. The field value consists of a single absolute URI.

Reply:
Ikkje iflg https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location#directives